### DISCUSSION TITLE
Just a quick question about speed

### DISCUSSION THREAD

**Discussion by Kademo15** [lshqqytiger/stable-diffusion-webui-directml#41] [Q&A] [answered]
Is there anybody that tested the speed of thd original automatic vs this fork vs the linux version on amd gpu 

If yes how much is the difference

info: i am on win11 with a rx7900xtx

**Comment by lshqqytiger** [comment, marked as answer]
Linux > this fork with `--opt-sub-quad-attention` >> this fork.
Original automatic's webui do not support AMD gpus on Windows.

**Reply by Kademo15** [reply]
Ok thx unfortunately i have a rx7900xtx and rdna3 is not yet supported by rocm so no linux for me.
